.storybook/stories/Video.stories.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+import { centered } from "./Decorators"
+
+const Video = (props) => {
+  return (
+    <video {...props}>
+      <source src="https://www.w3schools.com/html/mov_bbb.mp4" type="video/mp4" />
+      <source src="https://www.w3schools.com/html/mov_bbb.ogg" type="video/ogg" />
+      Your browser does not support HTML5 video.
+    </video>
+  )
+}
+
+export default {
+  title: 'HTML Elements/Video',
+  component: Video,
+  decorators: centered
+};
+
+// Safety first.
+const Template = (args) => <Video {...args} />
+
+export const Controls = Template.bind({})
+Controls.args = {
+  className: '',
+  controls: true,
+  autoplay: false,
+  loop: false,
+  muted: false,
+  preload: 'none',
+  poster: ''
+}
+
+export const NoControls = Template.bind({})
+NoControls.args = {
+  className: '',
+  controls: false,
+  autoplay: true,
+  loop: false,
+  muted: false,
+  preload: 'none',
+}
```

Comment:
Looks like if doing it this way and use the ...props cause the video/audio to go away completely if the controls are set to false.

Reply:
https://stackoverflow.com/questions/9565363/video-element-disappears-in-chrome-when-not-using-controls

Seems to be a bug in chrome

Reply:
Yikes 🤦🏼‍♀️

Reply:
Looks like you set some other attributes to true which works better. I'm looking at the video with no controls here (the ocean one) and that one works too, so maybe it needs one of those attributes set to true.
https://decanter.stanford.edu/component/simple-media/